packages/infra/client/isolate-v8-runner/js/90_rivet_ns.js

```diff
@@ -3,6 +3,42 @@
 // Generated with scripts/pegboard/compile_bridge.ts
 
 import { KV_NAMESPACE } from "ext:rivet_kv/40_rivet_kv.js";
+import { env } from "ext:runtime/30_os.js";
+function parseMetadata() {
+    let envMetadata = env.get("RIVET_METADATA");
+    let metadata = null;
+    if (envMetadata) {
+        try {
+            let parsedMetadata = JSON.parse(envMetadata);
+            if (parsedMetadata) {
+                metadata = {
+                    id: parsedMetadata.id,
+                    envId: parsedMetadata.env_id,
+                    datacenterId: parsedMetadata.datacenter_id,
+                    clusterId: parsedMetadata.cluster_id,
+                    region: parsedMetadata.region,
+                    tags: parsedMetadata.tags,
+                };
+            }
+        }
+        catch (e) {
+            console.warn("Rivet: failed to parse actor metadata");
+        }
+    }
+    // Fallback to defaults
+    if (!metadata) {
+        metadata = {
```

Comment:
should this be frozen?

Reply:
What does that mean?

Reply:
see `Object.freeze`. realizing this is probably already done by `core.propReadOnly`. is that correct?